apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -30,6 +31,24 @@ export default function HomeScreen() {
 }
 
 function HomeScreenInner({ account }: { account: Account }) {
+  const bottomSheetRef = useRef<SwipeUpDownRef>(null);
+  const nav = useNav();
+  const isFocused = useIsFocused();
+
+  useEffect(() => {
+    if (nav.getParent()) {
+      // @ts-ignore
```

Comment:
all equal i recommend local fixes (eg `e: any`) vs @ts-ignore for the whole line